wordpress-mobile/gutenberg-mobile
# Show a notice when user taps to add a disabled block option

* Gutenberg PR: https://github.com/WordPress/gutenberg/pull/35021
* WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/15435
* WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/17293

Fixes #3055

### Description
Whenever a block doesn't support multiple instances and is already added to
the post, its icon is greyed out and it can't be added. With this commit,
when the user taps on it, it shows a Notice with the information on why
that block can't be added.

### To test
1. Add a "More" block to the Post
2. Tap the "+" button to add a block
3. See that the "More" button is greyed out
4. Tap on the "More" button and see that a Notice is displayed on top 

PR submission checklist:
